ast/compile.go

```diff
@@ -1259,12 +1259,14 @@ func (c *Compiler) compile() {
 		if c.Failed() {
 			return
 		}
-		for _, s := range c.after[s.name] {
-			err := c.runStageAfter(s.MetricName, s.Stage)
-			if err != nil {
+		for _, a := range c.after[s.name] {
```

Comment:
Didn't appreciate the shadowing, changed the name. No consequences from this, though.

Reply:
This looks safe to me. :+1: